src/components/Player/Player.tsx

```diff
@@ -57,6 +58,10 @@ const Player: React.FC<Props> = ({
   const scriptUrl = `${import.meta.env.APP_API_BASE_URL}/libraries/${playerId}.js`;
   const setPlayer = useOttAnalytics(item, feedId);
 
+  const { adSchedule, features, custom } = useConfigStore((s) => s.config);
+  const recommendationsPlaylist = features?.recommendationsPlaylist;
+  const enableSharing: boolean = isTruthyCustomParamValue(custom?.enableSharing);
```

Comment:
It would probably be nice long term to convert these truthy param checks one time when the config is loaded and then just have the value in the config store be a boolean

Reply:
Yeah, good idea. Removed this param now but can be still done in the future.